src/test/resources/DoEarlyReturnWhenPossible.java

```diff
@@ -43,4 +43,30 @@ public String getValue(Element parentElem, String name, boolean mandatory) { //
     }
     return valueElement.getText();
   }
+
+  /*
+    `getReport` method from `InfComision` class. Found in 'Drivania' project. Testing rule's behaviour when the conditional 'else'
+    statement to be analyzed is a single expression, instead of a block of code (enclosed in braces)
+   */
+  private String getReport(ConnectionProvider cp, VariablesSecureApp vars, String strDateFrom, String strDateTo,
+      String strcBpartnerId, String strOrg, String strAgentId) throws Exception {
+    String partner = (strcBpartnerId == null || strcBpartnerId.isEmpty()) ? null : strcBpartnerId;
+    String agent = (strAgentId == null || strAgentId.isEmpty()) ? null : strAgentId;
+    String org = (strOrg == null || strOrg.isEmpty()) ? null :   strOrg;
+
+    PrintInvoicesData[] data = null;
+    if(null!= agent) {
```

Comment:
This doesn't have the [Etendo code style](https://docs.etendo.software/latest/developer-guide/etendo-classic/getting-started/installation/intellij-code-formatting/?h=code+style)